### PR TITLE
KZOO-343: Add endpoint call_limits merge tests

### PR DESCRIPTION
**Pull request by harenson**

### CODE REVIEW

File: kazoo_endpoint/test/kz_endpoint_v5_tests.erl [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+           end,
+    AccountLimit = {kzd_accounts:type(), kz_term:rand_integer(1, 10)},
+    OwnerLimit = {kzd_users:type(), kz_term:rand_integer(1, 10)},
+    EndpointLimit = {kzd_devices:type(), kz_term:rand_integer(1, 10)},
+    NotLimit = new_limited_entity(),
+    Account = new_limited_entity(AccountLimit),
+    Owner = new_limited_entity(OwnerLimit),
+    Endpoint = new_limited_entity(EndpointLimit),
+    Resp = Test(Account, NotLimit, NotLimit),
+    ?debugFmt("Resp: ~p", [Resp]), %% TODO: Remove this debug line
```
Comment (nkarunarathne) [inline kazoo_endpoint/test/kz_endpoint_v5_tests.erl]: I think these 2 lines need to be removed

Comment (harenson) [reply kazoo_endpoint/test/kz_endpoint_v5_tests.erl]: You are correct. I just removed those.

**Comment by nkarunarathne** [comment]
@harenson please change the copyright version.

**Comment by nkarunarathne** [comment]
This looks fine to me.

**Review by nkarunarathne — APPROVED** [review]
This looks good

**Pull request merged**